src/main/java/uk/gov/hmcts/reform/civil/handler/callback/camunda/notification/ClaimContinuingOnlineApplicantPartyForSpecNotificationHandler.java

```diff
@@ -72,9 +74,22 @@ RESPONSE_DEADLINE, formatLocalDate(caseData.getRespondent1ResponseDeadline()
     private void generateEmail(CaseData caseData) {
         notificationService.sendMail(
             caseData.getApplicant1Email(),
-            notificationsProperties.getClaimantClaimContinuingOnlineForSpec(),
+            getEmailTemplate(caseData),
             addProperties(caseData),
             String.format(REFERENCE_TEMPLATE, caseData.getLegacyCaseReference())
         );
     }
+
+    private boolean isBilingualForLipvsLip(CaseData caseData) {
```

Comment:
Can we add this function in caseData if it does not exist yet? Possibly it could be reused by other tickets, especially those concerning notifications.

Reply:
As discussed, It is good to add this method in caseData as this will require featureToggleService to be added as dependency in model.